moha/hamiltonians.py

```diff
@@ -89,6 +90,7 @@ def __init__(
         self.gamma = gamma
         self.charges = charges
         self.atom_types = None
+        self.dist_atoms = None
```

Comment:
just to be consistent with the name `atom_types` let's change the name `dist_atoms` to `atom_dists`